### PR TITLE
Update visualization_3d.py

### PR DESCRIPTION
**Pull request by RichardScottOZ**
Visualize

# Description
Please include a summary of the changes.

Relates to <issue>

# Checklist
- [ ] My code follows the [PEP 8 style guidelines](https://www.python.org/dev/peps/pep-0008/).
- [ ] My code uses type hinting for function and method arguments and return values.
- [ ] My code contains descriptive and helpful docstrings 
which are formatted per the [Google Python Style Guidelines](http://google.github.io/styleguide/pyguide.html).
- [ ] I have created tests which entirely cover my code.
- [ ] The test code either 1. demonstrates at least one valuable use case (e.g. integration tests) 
or 2. verifies that outputs are as expected for given inputs (e.g. unit tests).
- [ ] New and existing tests pass locally with my changes.

### CODE REVIEW

File: gempy/core/data.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ def __init__(self, project_name='default_project'):
 class Grid(object):
     """ Class to generate grids.
 
-    This class is used to create points where to
-    evaluate the geological model. This class serves a container which transmit the XYZ coordinates to the
-    interpolator. There are several type of grids objects will feed into the Grid class
+    This class is used to create points usede to
```
Comment (AlexanderJuestel) [inline gempy/core/data.py:56]: @RichardScottOZ could you change your correction here please. I would then merge it! Thanks :)

**Review by AlexanderJuestel — CHANGES_REQUESTED** [review]
@RichardScottOZ just change this one error and then we are finally good to go :)

File: gempy/core/data.py (hunk 1 above)
Comment (RichardScottOZ) [reply gempy/core/data.py:56]: oops, ok!

**Comment by AlexanderJuestel** [comment]
Thanks for the changes, merging now

**Pull request merged**